packages/esm-patient-medications-app/src/api/order-config.ts

```diff
@@ -57,16 +72,24 @@ export function useOrderConfig(): {
           valueCoded: uuid,
           value: display,
         })),
-        orderFrequencies: data?.data?.orderFrequencies?.map(({ uuid, display }) => ({
-          valueCoded: uuid,
-          value: display,
-        })),
+        orderFrequencies: frequencyData?.data?.orderFrequencies?.map(({ uuid, display, concept }) => {
+          const getAbbreviation = (names: Names[]) => {
+            return names.map((name) => name.display);
+          };
+
+          const abbreviation = getAbbreviation(concept.names);
+
+          return {
+            valueCoded: uuid,
+            value: display,
+            abbreviation: abbreviation,
```

Comment:
```suggestion
          return {
            valueCoded: uuid,
            value: display,
            names: concept.names.map(name => name.display),
```

An array should generally have a pluralized name, so `abbreviations` rather than `abbreviation`. But also this isn't just the abbreviations. This is all the names. So let's just call it names. Please adjust the rest of the code accordingly.